source/B2BApi.AppTests/Functions/EnqueueMessages/BRS_028/EnqueueBrs028MessagesTests.cs

```diff
@@ -159,7 +159,7 @@ public async Task Given_EnqueueAcceptedBrs028Message_AndGiven_GridAreaHasNoData_
         // Verify that the expected notify message was sent on the ServiceBus
         var notifyMessageSent = await ThenNotifyOrchestrationInstanceWasSentOnServiceBus(
             orchestrationInstanceId,
-            RequestCalculatedWholesaleServicesNotifyEventsV1.EnqueueActorMessagesCompleted);
+            RequestCalculatedWholesaleServicesNotifyEventV1.OrchestrationInstanceEventName);
         notifyMessageSent.Should().BeTrue("Notify EnqueueActorMessagesCompleted service bus message should be sent");
```

Comment:
```suggestion
        notifyMessageSent.Should().BeTrue("Notify EnqueueActorMessagesCompleted service bus message should be sent");
```
This "EnqueueActorMessagesCompleted" still the event witch the process manager receives?